discovery_engine_api_tests/handlers/api_handler.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+import requests
+import allure
+import logging
+from config.config import Config
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ApiHandler:
+    TIMEOUT = 5
+
+    def __init__(self):
+        conf = Config()
+        self.interactions = conf.get_interactions_endpoint()
+        self.ingestion = conf.get_ingestion_endpoint()
+
+    def post_document(self, doc):
```

Comment:
:art: This should be named after what it does, not after what is uses to do so.

E.g. `ingest_document`.